include/hal/library/responder_secretlib.h

```diff
@@ -138,6 +138,33 @@ extern bool libspdm_generate_measurement_summary_hash(
     size_t *measurement_summary_hash_size);
 #endif /* LIBSPDM_ENABLE_CAPABILITY_MEAS_CAP */
 
+#if LIBSPDM_ENABLE_CAPABILITY_CHAL_CAP
+/**
+ * This functions returns the opaque data in a CHALLENGE_AUTH response.
+ *
+ * @param spdm_version  Indicates the negotiated s version.
+ *
+ * @param  slot_id       The number of slot for the certificate chain.
+ *
+ * @param  measurement_hash_algo  Indicates the measurement hash algorithm.
+ * Must be SPDM_ALGORITHMS_MEASUREMENT_HASH_ALGO_* value in spdm.h.
+ *
+ * @param opaque_data
+ * A pointer to a destination buffer whose size, in bytes, is opaque_data_size. The opaque data is
+ * copied to this buffer.
+ *
+ * @param opaque_data_size
+ * On input, indicates the size, in bytes, of the destination buffer.
+ * On output, indicates the size of the opaque data.
+ **/
+extern bool libspdm_challenge_opaque_data(
```

Comment:
This is also needed in `requester_secretlib.h`.

Reply:
Yes. I have added it. Thanks.